src/entities/session/model/store.ts

```diff
@@ -0,0 +1,15 @@
+import { createEffect, createStore, sample } from 'effector';
+import { createGate } from 'effector-react';
+import { getCurrentUser, login } from './api';
+import { User } from './types';
+
+export const gate = createGate();
+
+export const loginFx = createEffect(login);
+const getCurrentUserFx = createEffect(getCurrentUser);
+
+export const $user = createStore<User | null>(null);
```

Comment:
entity should not have samples and stores. the only case when the store is global.

In this particular case store could be global, as user info from the session is may be needed in most of the parts of the application.

please try the next approach, just for understanding how to work with a local store: 
1. The store should be moved to the model of the responsible page
2. The sessionModel exports only needed effects via public API
3. The model of the responsible page imports the entity model
4. The event is called not directly from the sessionModel, but it is called locally, which means the request will be made from its own instance of the effect